### ISSUE TITLE
Failed to Install add-on

### ISSUE THREAD

**Issue by Jasonrobot00** [TECH7Fox/asterisk-hass-addons#260]
Hey, I'm in the process of moving y set up from a Raspberry Pi 4 to a NUC, I'm trying to set up Asterisk's again on my NUC however every time I try to install the add on I get the following error:

"Can't install ghcr.io/tech7fox/asterisk-hass-addon:4.0.4: 500 Server Error for http+docker://localhost/v1.42/images/create?tag=4.0.4&fromImage=ghcr.io%2Ftech7fox%2Fasterisk-hass-addon&platform=linux%2Famd64: Internal Server Error ("manifest unknown")"

I have Deleted and re-added the repository with no luck. Is anyone having the same issue, or know what the fix might be?
Also is there another 'manual' way to add the Asterisk's add-on ?

Thank you in advance.

**Comment by TECH7Fox** [comment]
Yes this is a known issue. See https://github.com/TECH7Fox/asterisk-hass-addons/issues/257

For now you could download the previous version and set the asterisk folder in your add-ons folder.

**Comment by felipecrs** [comment]
@TECH7Fox maybe you can revert version in config.yaml to previous version until this issue gets fixed.

I'm traveling in the next 5 days so I can't help to fix it :(

**Comment by TECH7Fox** [comment]
Changed to 4.0.3 until the issue is fixed.
